sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/geckoview_version_v1/init.sql

```diff
@@ -0,0 +1,147 @@
+CREATE TABLE IF NOT EXISTS
+  org_mozilla_fenix_derived.geckoview_version_v1(
+    build_hour DATETIME,
+    geckoview_version STRING,
+    n_builds INT64,
+  )
+AS
+WITH extracted AS (
+  -- We'll look at the metrics ping to estimate the major geckoview version.
+  -- The metrics section is aliased, so we must rename the table for this to
+  -- work as expected.
+  SELECT
+    submission_timestamp,
+    client_info.app_build,
+    metrics.string.geckoview_version,
+  FROM
+    org_mozilla_fenix.metrics AS t1
+  WHERE
+    mozfun.norm.fenix_app_info('org_mozilla_fenix', client_info.app_build).channel = 'nightly'
+  UNION ALL
+  SELECT
+    submission_timestamp,
+    client_info.app_build,
+    metrics.string.geckoview_version,
+  FROM
+    org_mozilla_fenix_nightly.metrics AS t1
+  UNION ALL
+  SELECT
+    submission_timestamp,
+    client_info.app_build,
+    metrics.string.geckoview_version,
+  FROM
+    org_mozilla_fennec_aurora.metrics AS t1
+),
+transformed AS (
+  SELECT
+    app_build,
+    geckoview_version,
+    -- Truncate to the hour, since older builds give minute resolution.
+    datetime_TRUNC(mozfun.norm.fenix_build_to_datetime(app_build), HOUR) AS build_hour
+  FROM
+    extracted
+  WHERE
+    DATE(submission_timestamp)
+    BETWEEN DATE_SUB(@submission_date, INTERVAL 30 DAY)
+    AND @submission_date
+),
+grouped_build_hours AS (
+  -- Count the number of geckoview versions for each build hour row over the
```

Comment:
Doesn't this count the number of pings per build_hour and geckoview_version? Not geckoview versions for each build hour.  If it is n_builds would be the wrong name.  But I think counting number of pings sounds reasonable.